examples/dfdl-firestore-pubsub-example/README.md

```diff
@@ -0,0 +1,188 @@
+# DFDL Processor Example
```

Comment:
What is DFDL? Could the acronym be expanded in this README?

Reply:
Done